run_dir/static/js/qpcr_pools.js

```diff
@@ -54,7 +54,13 @@ function load_table() {
             });
             return to_return;
           }));
-          tbl_row.append($('<td>').html((avg_wait_calc/pools['samples'].length).toFixed(1)));
+          var lblinf='';
+          switch(Math.floor((avg_wait_calc/pools['samples'].length)/7)){
```

Comment:
Why are you dividing the average wait time by the number of samples?

Reply:
Its really a counter for calculating the average rather than the average itself . My bad with the nomenclature😬